modules/ROOT/pages/policies-policy-level.adoc

```diff
@@ -8,52 +8,50 @@ Policies are by default applied to the entire API, unless configured otherwise.
 
-image::methods-resources.png[]
+//image::methods-resources.png[]
 
 You can apply multiple conditions to filter your resources and HTTP methods. The URI template regex can be applied to one, many, or all the methods in your API.
 
-== Usage Scenarios
+//explain here what is URI template regex--fede
 
-The uses for resource level policies are limitless. Listed below are a few examples of how you might implement the resource-level policies:
-The uses for resource-level policies are limitless. A few examples are:
+== Usage Scenarios for Resource-Level Policies
+
+The uses for resource-level policies are limitless. Listed below are a few examples of how you might implement the resource-level policies:
 
 * Applying policies to specific resources
 * Securing a subset of an API
 * Setting different limits on resources
 
-If you have a RAML or OAS spec attached to your API, click Preview Resource Matching to check which resources your filters affect.
-
-//Where does a user click this?
-
-Following is an example API for the usage scenario:
+If you have a RAML or OAS specification attached to your API, you can use the  Preview Resource Matching option to check which resources are affected by your filters:
 
 image::users-api-cropped.png[API Console displays resources and methods,height=329,width=732]
 
 A resource-level policy supports regular Java expressions. For example, you can use the wildcard to apply a policy to multiple resources. When you apply the policy to the API, specify the resources to which it applies.
 
-// were there differences between mule 3 and mule 4 in the expressions? if there are we should highlight them here -- Fede Balbi?
 
-image::rlp-conditions.png[]
+//image::rlp-conditions.png[]
 
-Do not use a placeholder, such as {userid}, in the regular expression. Using a placeholder in an expression fails because the placeholder does not match the actual node. In the example placeholder {userid}, the node looks something like this:
+== Guidelines for Implementing Resource-Level Policies
 
-Example: 
-`/api/users/671962fc-f076-4b19-bc38-45ba3a4e4095/permissions/1234`
+When implementing resource-level policies:
 
-* 671962fc-f076-4b19-bc38-45ba3a4e4095 is the user ID
-* 1234 is the ID of a permission
-
-To apply a policy to resource `/api/users/{userid}` that represents a single user and all resource nodes, use the following expression:
+* Do not use a placeholder, such as {userid}, in the regular expression. 
++
+Using a placeholder in an expression fails because the placeholder does not match the actual node. In the example placeholder {userid}, the node looks something like this:
++
+`/api/users/671962fc-f076-4b19-bc38-45ba3a4e4095/permissions/1234`
 
+** `671962fc-f076-4b19-bc38-45ba3a4e4095` is the user ID
```

Comment:
Does your audience really require an example of why we suggest this guideline?

Reply:
Yes, examples are always good. Done! Thanks!